cxx-qt-gen/src/parser/cxxqtdata.rs

```diff
@@ -335,6 +339,55 @@ mod tests {
             .is_some());
     }
 
+    #[test]
+    fn test_find_and_merge_cxx_qt_item_struct_valid_properties() {
+        let mut cxx_qt_data = create_parsed_cxx_qt_data();
+
+        let item: Item = tokens_to_syn(quote! {
+            #[cxx_qt::qobject]
+            struct MyObject {
+                #[property]
+                property: i32,
+            }
+        });
+        let result = cxx_qt_data.parse_cxx_qt_item(item).unwrap();
+        assert!(result.is_none());
+        assert_eq!(cxx_qt_data.qobjects[&qobject_ident()].properties.len(), 1);
+    }
+
+    #[test]
+    fn test_find_and_merge_cxx_qt_item_struct_valid_properties_and_fields() {
+        let mut cxx_qt_data = create_parsed_cxx_qt_data();
+
+        let item: Item = tokens_to_syn(quote! {
+            #[cxx_qt::qobject]
+            struct MyObject {
+                #[property]
+                property: i32,
+
+                field: i32,
+            }
+        });
+        let result = cxx_qt_data.parse_cxx_qt_item(item).unwrap();
+        assert!(result.is_none());
+        assert_eq!(cxx_qt_data.qobjects[&qobject_ident()].properties.len(), 1);
```

Comment:
add assertions for the ParsedProperty's ident/type/visibility fields

Reply:
Done

Reply:
Moved these assertions into the qobject parse_struct_fields tests, then the global tests just check that properties were found rather than duplicating the tests.